Userland/Libraries/LibCore/FileSystem.cpp

```diff
@@ -231,15 +240,16 @@ ErrorOr<void, CopyError> copy_file(DeprecatedString const& dst_path, DeprecatedS
             src_stat.st_mtim,
 #endif
         };
-        if (utimensat(AT_FDCWD, dst_path.characters(), times, 0) < 0)
+        DeprecatedString dst_path_string = dst_path;
+        if (utimensat(AT_FDCWD, dst_path_string.characters(), times, 0) < 0)
```

Comment:
`utimensat` is one of the chonkier LibC syscall wrappers, but we could technically add a `Core::System::utimensat` that takes a `StringView` and sends the string to the kernel without copying.